catalog/admin/module/catalog-admin-module-sources/src/main/webapp/js/model/Source.js

```diff
@@ -133,6 +135,27 @@ define([
 
                 return actions;
             },
+            getCurrentUrl: function () {
+                var src = this;
+                var currentConfig = src.get('currentConfiguration');
+                var configProps = currentConfig.attributes.properties.attributes;
+                var configPropKeys = Object.keys(configProps);
+
+                var urls = configPropKeys.filter(function(item) {
+                    return /.*Address|.*Url/.test(item);
+                });
+                var filteredKeys = urls.filter(function (item) {
+                    return /^(?!event|site).*$/.test(item);
```

Comment:
Any way we could use constant values and avoid filtering the keys and URLs using regex? Those always leave the door open to potential problems down the road. The other, better alternative would be to have the backend code tell the UI when a URL is a loopback instead of having the UI do all this work, but that's probably not worth the effort considering that this is a temporary solution.